mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
+        return res.status(404).json({ error: "No such order found" });
+      }
+      res.json({ success: "Order has been deleted" });
+    })
+    .catch(error => {
```

Comment:
Inconsistent error handling. The error handler function `errorHandler` is not used here, unlike in other parts of the code. It should be used to handle errors consistently throughout the codebase.